docs/micro-rdk/board/esp32.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Configure an ESP32 board"
+linkTitle: "esp32"
+weight: 20
+type: "docs"
+description: "Configure an esp32 board."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components"]
+# SMEs: Gautham, Rand
+---
+
+{{% alert title="REQUIREMENTS" color="caution" %}}
+
+Follow the [setup guide](/installation/prepare/microcontrollers/) to prepare your ESP32 for running micro-RDK before configuring a `esp32` board.
+
+{{% /alert %}}
+
+To add an `esp32` board, navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com) and select **Raw JSON** mode.
+
+Copy the following JSON template and paste it into your configuration inside the `"components"` array:
+
+{{< tabs name="Configure an esp32 Board" >}}
+{{% tab name="JSON Template"%}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "attributes": {
+    "pins": [
+      <int>
+    ],
+    "analogs": [
+      {
+        "pin": "<number>",
+        "name": "<your-analog-name>"
+      }
+    ]
+  },
+  "depends_on": [],
+  "model": "esp32",
+  "name": "<your-board-name>",
+  "type": "board"
```

Comment:
Move to top of config and add namespace if applicable. I've been ordering everything name, model, type, namespace, attributes, depends_on for consistency, and also because that's how the app seems to autopopulate it these days, and also because I think name model and type make a lot more sense at the beginning, to sort of introduce the component, before getting into the details.

Reply:
That makes a lot of sense, done! TY!